core/domain/stats_jobs_continuous_test.py

```diff
@@ -452,7 +451,7 @@ def test_one_answer(self):
             # get job output of first state and check it
             calc_output_domain_object = (
                 stats_jobs_continuous.InteractionAnswerSummariesAggregator.get_calc_output(
-                    exp_id, exp_version, FIRST_STATE_NAME, calc_id))
+                    exp_id, FIRST_STATE_NAME, calc_id, exp_version))
```

Comment:
Should it be exploration_version=exp_version?

Alternatively, scrap the default value and require it to be passed in explicitly. It doesn't seem unreasonable to do that...